files/build/templates/buildinfo.config.j2

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+BUILDINFO_PATH=/usr/local/share/buildinfo
+
+LOG_PATH=$BUILDINFO_PATH/log
+VERSION_PATH=$BUILDINFO_PATH/versions
+PRE_VERSION_PATH=$BUILDINFO_PATH/pre-versions
+DIFF_VERSION_PATH=$BUILDINFO_PATH/diff-versions
+BUILD_VERSION_PATH=$BUILDINFO_PATH/build-versions
+POST_VERSION_PATH=$BUILDINFO_PATH/post-versions
+VERSION_DEB_PREFERENCE=$BUILDINFO_PATH/versions/01-versions-deb
+
+PACKAGE_URL_PREFIX={{ PACKAGE_URL_PREFIX }}
```

Comment:
>PACKAGE_URL_PREFIX [](start = 22, length = 18)

What is the benefit of using J2 template? How about using environment variable and remove all the indirect levels regarding rendering template?
Dynamic code is more flexible and readable then rendering.

Reply:
The j2 files Removed